help/en/releasenotes/current-draft-note.shtml

```diff
@@ -598,6 +598,8 @@
     <h3>Miscellaneous</h3>
         <a id="Misc" name="Misc"></a>
         <ul>
-            <li></li>
+            <li>The
+                <a href="https://www.jmri.org/help/en/html/doc/Technical/Patterns.shtml#typescript">Typescript and Javascript support</a>
+                has been updated.</li>
```

Comment:
I suggest this is changed to
```
<li>Note for JMRI developers:<br>
The
<a href="https://www.jmri.org/help/en/html/doc/Technical/Patterns.shtml#typescript">Typescript and Javascript support</a>
has been updated.</li>
```
since the users aren't affected by this and they might be confused by this note.